src/dispatch/config.py

```diff
@@ -190,24 +183,35 @@ def __str__(self) -> str:
 INCIDENT_RESOURCE_NOTIFICATIONS_GROUP = config(
     "INCIDENT_RESOURCE_NOTIFICATIONS_GROUP", default="google-group-participant-notifications-group"
 )
+INCIDENT_RESOURCE_INVESTIGATION_SHEET_TEMPLATE = config(
+    "INCIDENT_RESOURCE_INVESTIGATION_SHEET_TEMPLATE", default="dispatch-incident-sheet-template"
+)
+INCIDENT_RESOURCE_INCIDENT_REVIEW_DOCUMENT_TEMPLATE = config(
+    "INCIDENT_RESOURCE_INCIDENT_REVIEW_DOCUMENT_TEMPLATE",
+    default="dispatch-incident-review-document-template",
+)
+INCIDENT_RESOURCE_EXECUTIVE_REPORT_DOCUMENT_TEMPLATE = config(
+    "INCIDENT_RESOURCE_EXECUTIVE_REPORT_DOCUMENT_TEMPLATE",
+    default="dispatch-executive-report-document-template",
+)
 INCIDENT_RESOURCE_INVESTIGATION_DOCUMENT = config(
-    "INCIDENT_RESOURCE_INVESTIGATION_DOCUMENT", default="google-docs-investigation-document"
+    "INCIDENT_RESOURCE_INVESTIGATION_DOCUMENT", default="dispatch-incident-document"
```

Comment:
Changing the default value will break older incidents, as Dispatch won't be able to locate the documents using the resource type. Are you planning to update them?

Reply:
Yeah, we can write a migration task.